### ISSUE TITLE
run_gui not working 

### ISSUE THREAD

**Issue by davidgwyrick** [ackmanlab/pySEAS#4]
Hello! Not sure if you check this anymore, but I was wondering if you had any quick insights as to why I get the following error when I try to run the gui with the ICA results. I'm using your code in kind of a very light way. To get the ICA results dictionary, I first ran 

#dfof_flattened is of size [160000, 9600] 
#According to your format (t,x,y), the second input is the size; seems like  backwards given the order of the flattened movie shape
`results = ica.project(dfof_flattened, (nFrames, nHeight, nWidth), brainmask, 500)`

then when I run the view_components script, pointing to the correct file containing the dictionary, I get. 

`Traceback (most recent call last):
  File "view_components.py", line 31, in <module>
    run_gui(args['input'][0].name,
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 1558, in run_gui
    app = PCAgui()
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 264, in __init__
    frame = F(container, self)
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 371, in __init__
    self.loadPCpage()
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 530, in loadPCpage
    self.PCfigure = self.PCframe(self, PCindices)
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 396, in __init__
    self.imagebutton.append(self.imButton(frame, n))
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 438, in __init__
    self.update(pc_id)
  File "/home/dwyrick/Git/pySEAS/seas/gui.py", line 459, in update
    self.imgplot = self.ax.imshow(eigenbrain,
  File "/home/dwyrick/miniconda3/envs/pyseas/lib/python3.8/site-packages/matplotlib/_api/deprecation.py", line 454, in wrapper
    return func(*args, **kwargs)
  File "/home/dwyrick/miniconda3/envs/pyseas/lib/python3.8/site-packages/matplotlib/__init__.py", line 1423, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/home/dwyrick/miniconda3/envs/pyseas/lib/python3.8/site-packages/matplotlib/axes/_axes.py", line 5604, in imshow
    im.set_data(X)
  File "/home/dwyrick/miniconda3/envs/pyseas/lib/python3.8/site-packages/matplotlib/image.py", line 710, in set_data
    raise TypeError("Invalid shape {} for image data"
TypeError: Invalid shape (500, 400, 400) for image data`

**Comment by scweiser** [comment]
Hi David,

Thank you for the bug report!

Sorry for the late reply, I just got access again to this email.
It seems like a small bug was introduced in the rebuild_eigenbrain function, where the call accidentally assigned the masind input to the assign_bulk parameter, so many frames were rebuilt instead of just one.

Sorry about that-- hope it wasn't too much of an inconvenience.

I just pushed an update now which fixes it.  I'm in the middle of another round of cleanup and documentation which should catch bugs like these, and will push to pypi soon.
